plugins/action/generate_cloud_modules.py

```diff
@@ -1339,6 +1844,70 @@ def generate_vmware_rest(args: Iterable, role_path: str):
     return
 
 
+def generate_cisco_intersight(args: Iterable, role_path: str):
+    module_list = []
+
+    for json_file in ["intersight_server.json"]:
```

Comment:
Could you pick up the file name from modules.yaml?

Reply:
I was following VMware's example since we'll have 100+ modules in a single schema file, but only a few schema files to load.  Looks like AWS has more of a single module to single schema file mapping.